src/XmlToLua.cpp

```diff
@@ -0,0 +1,792 @@
+#include "global.h"
+#include "ActorUtil.h"
+#include "Foreach.h"
+#include "IniFile.h"
+#include "RageFile.h"
+#include "RageFileManager.h"
+#include "RageLog.h"
+#include "RageTypes.h"
+#include "LuaManager.h"
+#include "LuaBinding.h"
+#include "XmlFile.h"
+#include "XmlFileUtil.h"
+
+#include <vector>
+#include <map>
+#include <set>
+
+#define TWEEN_QUEUE_MAX 50
+
+RString unique_name();
```

Comment:
Not a good reason.  I'll add one and get rid of that out of place prototype in ScreenDebugOverlay.cpp.